extensions/wikia/WikiaHubsServices/modules/WikiaHubsModuleFromthecommunityService.class.php

```diff
@@ -259,21 +260,24 @@ public function getStructuredData($data) {
 		$entries = array();
 		for($i = 1; $i <= $boxesCount; $i++) {
 			if( $this->isEntryFilledIn($i, $data) ) {
-				if( !empty($data['photo' . $i]) ) {
-					$imageData = $this->getImageInfo($data['photo' . $i]);
-				}
-
 				$entries[] = array(
 					'articleTitle' => $data['title' . $i],
 					'articleUrl' => str_replace(' ', '_', $data['url' . $i]),
-					'imageAlt' => empty($imageData->title) ? null : $imageData->title,
-					'imageUrl' => empty($imageData->url) ? null : $imageData->url,
 					'userName' => $data['UserName' . $i],
 					'userUrl' => str_replace(' ', '_', $data['usersUrl' . $i]),
 					'wikiUrl' => $data['wikiUrl' . $i],
 					'quote' => $data['quote' . $i],
 					'photoName' => isset($data['photo'.$i]) ? $data['photo'.$i] : ''
 				);
+
+				if( !empty($data['photo' . $i]) ) {
+					$imageData = $this->getImageInfo($data['photo' . $i]);
+
+					if ($imageData) {
+						$entries['imageAlt'] = $imageData->getName();
```

Comment:
I don't think this is what you want. You probably want to have `imageAlt` and `imageUrl` set for one specific entry from the `$entries`.